cluster/images/etcd-empty-dir-cleanup/Makefile

```diff
@@ -14,9 +14,9 @@
 
 .PHONY:	build push
 
-ETCD_VERSION = 2.2.1
+ETCD_VERSION = 3.0.14
 IMAGE = gcr.io/google_containers/etcd-empty-dir-cleanup
-TAG = 0.0.1
+TAG = 3.0.14
```

Comment:
Agreed, we need to come up with a convention for all our images. Semantic versioning calls for `3.0.14-whatever`. Also, this image is more than the etcd version, what happens when we modify the cleanup script? I suggest keeping the image version scheme (`0.1.0` or maybe `0.1.0-etcd3.0.14`)